bin/dynamodb-admin.js

```diff
@@ -38,9 +38,15 @@ parser.add_argument('-p', '--port', {
   help: 'Port to run on (default: 8001)',
 })
 
+parser.add_argument('-D', '--dummy_creds', {
+  action:'store_true',
+  required: false,
+  help: 'Use dummy AWS credentials',
+})
```

Comment:
For it not to be a breaking change, it would need to by default use dummy credentials.

Reply:
Yeah , I spent a bunch of time thinking about that.
Makes sense to default it to true